### PR TITLE
feat: export test with related properties/metadata

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**
When exporting test we will export also all resources metadata descriptions as metaMetadata objects in lom notation[*](https://www.imsglobal.org/node/52586#1186395)

Steps to test

1. Enable feature flag `FEATURE_FLAG_LOM_ONTOLOGY_EXTRACTION`
2. Add properties in items
3. Add properties in test
4. Add items with properties in test
5. Export test
6. Open and extract zip package
7. Preview imsmanifest.xml file. 
    - Ensure all properties are in place
    - Ensure no duplicates are present
    - Ensure xml file is valid

### CODE REVIEW

**Review by Karol-Stelmaczonek — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by tikhanovichA — APPROVED** [review]
Good job
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code respects code style rules
- [x] New code respects best practices
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic

**Review by Karol-Stelmaczonek — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: models/classes/metadata/ChecksumGenerator.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        if (empty($resourceList)) {
+            return '';
+        }
+        $labels = [];
+        foreach ($resourceList as $resource) {
+            $labels[] = strtolower($this->ontology->getResource($resource['id'])->getLabel());
+        }
+        asort($labels);
+        $checksum = implode('', $labels);
+        common_Logger::e(sprintf('ChecksumGenerator resource before sha1 : "%s"',  $checksum));
```
Comment (tikhanovichA) [inline models/classes/metadata/ChecksumGenerator.php:55]: 55 | ERROR | [x] Expected 1 space after comma in argument list; 2 found

**Comment by bartlomiejmarszal** [comment]
This code will be merged with: https://github.com/oat-sa/extension-tao-testqti/pull/2454

**Pull request closed without merging**